src/types/Task.types.ts

```diff
@@ -0,0 +1,5 @@
+export type Task = {
+  status: string;
+  title?: string;
```

Comment:
I think there shouldn't be any one of the props optional

When do you have a task without a label/title or without an id?
The id is the only unique thing you can use to retrieve the task.

Reply:
I have task without a label/title when I am marking a  task  as completed or deleting one. In those cases only the ID is needed.

I also don't need the ID when I am creating a task for the first time, at that time we need only the title & status. 

I thought about the problem you mention as well and was conflicted for how to set up the props which is why I set title & id as optional. How do you think I could go about it? This was the solution I thought was best in the moment, other than creating different types for the different states of tasks depending on the action needing to be taken against them. Didn't want too much repetition.

Reply:
Leave it that way for now. I'll show later how we can tackle the thing. Have faith young lady!

Reply:
Haha, ok will do, captain. :)